py/server/tests/test_table.py

```diff
@@ -1115,6 +1115,64 @@ def test_arg_validation(self):
             t.partition_by("A", "B")
         self.assertIn("drop_keys must be", str(cm.exception))
 
+    def test_table_diff(self):
+        with self.subTest("diff"):
+            t1 = empty_table(10).update(["A = i", "B = i", "C = i"])
+            t2 = empty_table(10).update(["A = i", "B = i % 2 == 0? i: i + 1", "C = i % 2 == 0? i + 1: i"])
+            d = table_diff(t1, t2, max_diffs=10).split("\n")
+            self.assertEqual(len(d), 3)
+            self.assertIn("row 1", d[0])
+            self.assertIn("row 0", d[1])
+
+            d = table_diff(t1, t2).split("\n")
+            self.assertEqual(len(d), 2)
+
+        with self.subTest("diff - ignore column order"):
+            t1 = empty_table(10).update(["A = i", "B = i + 1"])
+            t2 = empty_table(10).update(["B = i + 1", "A = i"])
+            d = table_diff(t1, t2, max_diffs=10).split("\n")
+            self.assertEqual(len(d), 3)
+
+            t1 = empty_table(10).update(["A = i", "B = i"])
+            t2 = empty_table(10).update(["B = i", "A = i"])
+            d = table_diff(t1, t2, max_diffs=10, ignore_column_order=True)
+            self.assertEqual(d, "")
+
+        with self.subTest("diff - floating_comparison = 'absolute' - double"):
```

Comment:
```suggestion
        with self.subTest("diff - floating_comparison = 'absolute'-double"):
```
Spacing is consistent with what is below.